ts/spi/provider_frame_test.ts

```diff
@@ -456,14 +456,74 @@ describe('ProviderFrame', () => {
          });
     });
 
+    // tests can use this to emulate user interaction in the credential picker
+    const pwdOrFbHintOptions: CredentialHintOptions = {
+      supportedAuthMethods: [
+        AUTHENTICATION_METHODS.ID_AND_PASSWORD,
+        AUTHENTICATION_METHODS.FACEBOOK
+      ]
+    };
+
+    describe('handling hintAvailable', () => {
+
+      it('should return false when the store is empty', async function(done) {
+        credentialDataProvider.credentials = [];
+
+        clientChannel.listen(
+            msg.RPC_MESSAGE_TYPES.hintAvailableResult, (res) => {
+              expect(res.args).toBe(false);
+              done();
+            });
+
+        clientChannel.send(
+            msg.hintAvailableMessage(requestId, pwdOrFbHintOptions));
+      });
+
+      it('should return false when the store is empty', async function(done) {
```

Comment:
This test is the same than the previous one. Remove, or update if it was meant to test something different.